s3.tf

```diff
@@ -0,0 +1,54 @@
+data "aws_iam_policy_document" "s3_for_github" {
+
+  statement {
+    sid    = "AllowListDescribe"
+    effect = "Allow"
+    actions = [
+      "s3:GetAccelerateConfiguration",
+      "s3:GetBucketAcl",
+      "s3:GetBucketLocation",
+      "s3:GetBucketLogging",
+      "s3:GetBucketNotificationConfiguration",
```

Comment:
I think this should be `s3:GetBucketNotification` rather than `s3:GetBucketNotificationConfiguration` - see [docs](https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetBucketNotificationConfiguration.html) `s3:GetBucketNotification` (although the API method has changed to `GetBucketNotificationConfiguration`).